dev-java/guice/guice-5.1.0.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+# Skeleton command:
+# java-ebuilder --generate-ebuild --workdir . --pom core/pom.xml --download-uri https://github.com/google/guava/archive/v30.1.1.tar.gz --slot 5 --keywords "~amd64 ~arm ~arm64 ~ppc64 ~x866" --ebuild guice-5.1.0.ebuild
+
+EAPI=8
+
+JAVA_PKG_IUSE="doc source test"
+MAVEN_ID="com.google.inject:guice:5.1.0"
+JAVA_TESTING_FRAMEWORKS="junit-4"
+
+inherit java-pkg-2 java-pkg-simple
+
+DESCRIPTION="Guice is a lightweight dependency injection framework for Java 6 and above"
+HOMEPAGE="https://github.com/google/guice"
+SRC_URI="https://github.com/google/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="4"
```

Comment:
Do we keep the slot? Or shoudn't it be '5' (probably using `SLOT=$(ver_cut 1)`)?

Reply:
> Do we keep the slot? Or shoudn't it be '5' (probably using `SLOT=$(ver_cut 1)`)?

Same slot. Otherwise we'd need to change reverse dep.  That's the reason for testing with 
```
`FEATURES=test` emerge -1a dom4j
```